dev_tools/scripts/db_integrity_test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Insert some test data in the db
+docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submission_file_path,submission_user,submission_file_extension,status,encryption_method) VALUES('test.c4gh','dummy','c4gh','INIT','CRYPT4GH');"
+
+# Build, run and execute the pg_basebackup action
+docker container run --rm -i --name pg-backup --network=host $(docker build -f dev_tools/Dockerfile-backup -q -t backup .) /bin/sda-backup --action pg_basebackup
+
+# Find the name of the copy in the S3 and check the length
+DBCOPY=$(s3cmd -c dev_tools/s3conf ls s3://dumps/ | grep ".tar" | cut -d '/' -f4)
+DBCOPYLENGTH=`echo -n "$DBCOPY" | wc -m`
+
+# Find the name of the backup image created in the previous step and check the length
+IMAGENAME=$(docker images -q backup)
+NAMELENGTH=`echo -n "$IMAGENAME" | wc -m`
+
+# Make sure that the image and the db copy exist before moving on
+if [ $NAMELENGTH = 0 ] || [ $DBCOPYLENGTH = 0 ]; then
+    echo "the image or the db copy is missing"
+    exit 1
+fi
+
+# Get the db copy from S3
+docker volume create restore
+docker container run --rm -i --name pg-backup --network=host -v restore:/home backup /bin/sda-backup --action pg_db-unpack --name "$DBCOPY"
+
+# Remove everything from the db
+docker exec -i db /bin/sh -c "rm -r data/pgdata"
+
+# Check that the pgdata are not there anymore
+USER=$(docker exec db psql -U postgres -d skata -tA -c "select submission_user from local_ega.main where submission_file_path = 'test.c4gh';") 2>/dev/null
+if [ "$USER" = "dummy" ]; then
+    echo "Failed to drop database"
+    exit 1
+fi
+
+# Stop the running db
+docker stop db
+
+# Add the physical copy in the db container
+docker run --rm -v restore:/pg-backup -v dev_tools_pgData:/pg-data alpine cp -r /pg-backup/db-backup/ /pg-data/pgdata/
+
```

Comment:
```suggestion
# Stop the running db
docker stop db

# Add the physical copy in the db container
docker run --rm -v restore:/pg-backup -v dev_tools_pgData:/pg-data alpine /bin/sh -c "rm -r data/pgdata && cp -r /pg-backup/db-backup/ /pg-data/pgdata/"

```
Stop the DB before deleting files to prevent false errors